src/ui/hexedit.cc

```diff
@@ -211,6 +218,25 @@ HexEdit::HexEdit(FileBlobModel *dataModel, QItemSelectionModel *selectionModel,
     copyMenu->addAction(copyAction);
   }
 
+  auto pasteMenu = menu_.addMenu("Paste as");
+
+  menu_.addSeparator();
```

Comment:
Please move the separators so that one is above "Save to file" and one is below "Paste as"